tests/test_0692_fsspec.py

```diff
@@ -29,7 +31,7 @@ def test_open_fsspec_github():
 
 @pytest.mark.network
 def test_open_fsspec_local(tmp_path):
-    url = "https://github.com/CoffeaTeam/coffea/raw/master/tests/samples/nano_dy.root"
+    url = skhep_testdata.data_path("uproot-issue121.root")
 
     # download file to local
```

Comment:
This test may now be redundant because `scikit-hep-testdata` has already downloaded the file.

Reply:
I was under the wrong impression that `skhep_testdata.data_path` would just return the full url for the file instead of a local path after downloading the file. Everything should be alright now.